### PR TITLE
Add myself (Patrik Svensson)

### PR DESCRIPTION
**Pull request by patriksvensson**
# #WeAreDonet Creator Submission Guidelines

* By submitting this pull request to add my creator card, I verify that I adhere to the #WeAreDotnet creator card submission guidelines.

* We do not accept submissions with the sole intention of promoting a product or company.

* We add you as a creator, not your company. 

* The moderator team decides whether we accept or reject a submission.

## When Do We Accept Someone's Creator Card Submission? 

* You must have published a post since 2022 on any of your channels. 

* There must be one publication in English. 

* There needs to be more than tweets. 

* A longer-form article on LinkedIn is good, but a self-hosted blog or one on a well-known platform like Hashnode, medium etc., is better. 

* Alternatively, you contributed with another form on a platform like YouTube, Twitch, Udemy or Pluralsight. If you have some recorded talks of conferences or anywhere else, that counts too! So in the form of a recorded video or a book. 

## RSS or Atom Feed 

* Submit an Atom or RSS URL inside your creator card that belongs to your blog 

* If you do not submit an RSS or Atom URL. We cannot consider your feed for the upcoming aggregation feature. 

## Content Guideline 

* If you have a self-hosted blog, please ensure you have a valid HTTPS certificate. 

* We host NO malicious or offensive content on the blog (including photos, swearing, etc.)

* You must add the full URL to your GitHub profile to your card, so we can assign it to the member account that used the same GitHub account for sign-up.

### CODE REVIEW

File: creator-cards/patrik-svensson.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  ],
+  "feeds": [
+    {
+      "type": "LinkTagAtomType",
+      "url": "https://patriksvensson.se/feed.xml"
+    },
+    {
+      "type": "AHrefFeedType",
+      "url": "https://patriksvensson.se/atom.xml"
```
Comment (DeeJayTC) [inline creator-cards/patrik-svensson.json:30]: If both feed types return the same results just give us one tho we don't have to filter dublicates

**Review by DeeJayTC — CHANGES_REQUESTED** [review]
Can you please give us a feed that's filtered for .NET and related content? I've seen a couple of Rust posts in the feed

**Comment by patriksvensson** [comment]
Ah, I'll remove the feed altogether then when I'm at a computer.

**Comment by patriksvensson** [comment]
Updated

**Comment by DeeJayTC** [comment]
So just double checking, you do not have a feed purely with .NET content and only want your channels listed?

**Comment by patriksvensson** [comment]
Correct. Not a one trick pony 😉

**Comment by patriksvensson** [comment]
No rush

**Comment by matthiasjost** [comment]
Any updates on this?

There is a new format allows one to have projects: https://github.com/WeAreDotnet/dotnet-creator-cards/blob/main/creator-cards/_creatorTemplate.json

**Comment by patriksvensson** [comment]
Ah, good to know. Will update with some projects.

**Review by arpitsdotnet — COMMENTED** [review]
Done

**Comment by matthiasjost** [comment]
@patriksvensson Any updates on this?

The new card format:
- https://github.com/WeAreDotnet/dotnet-creator-cards#sample-card
- https://github.com/WeAreDotnet/dotnet-creator-cards/blob/main/creator-cards/_creatorTemplate.json

The new settings page:
- https://www.wearedotnet.io/settings

It would be cool to have you onboard!

Greetings

Matthias

**Pull request closed without merging**